packages/commons/src/sqs/queueManagerMessageValidation.ts

```diff
@@ -0,0 +1,26 @@
+import { invalidSqsMessage } from "pagopa-interop-models";
+import { Logger } from "../logging/index.js";
+import { Message } from "./index.js";
+
+type EventValidation = "ValidEvent" | "InvalidEvent";
```

Comment:
this feels less like an unexpected error
```suggestion
type EventValidation = "ValidEvent" | "SkipEvent";
```